typescript/sdk/src/deploy/middleware/LiquidityLayerApp.ts

```diff
@@ -75,6 +78,44 @@ export class LiquidityLayerApp<
     return response.result.map((_: any) => _.transactionHash).flat();
   }
 
+  async fetchPortalBridgeTransactions(chain: Chain): Promise<string[]> {
+    const cc = this.multiProvider.getChainConnection(chain);
+    const params = new URLSearchParams({
```

Comment:
still feel like this should be in chain connection
is there no etherscan client?

Reply:
Agreed it should be, but there is not.

Filed
https://github.com/hyperlane-xyz/hyperlane-monorepo/issues/1325
https://github.com/hyperlane-xyz/hyperlane-monorepo/issues/1326